### ISSUE TITLE
Complete Wiki

### ISSUE THREAD

**Issue by Ulvtand** [ultrakorne/better-rolltables#136]
Wiki is substandard considering how much you have developed this module <3

**Comment by DanielBoettner** [comment]
You are not wrong about the wiki. But instead of remarking the obvious you could try to help :P

Any suggestions on what should be documented with priority?

**Comment by Ulvtand** [comment]
Well, I'm dyslexic and do not have English as my mother tongue. But of course I can help but someone has to go through the text to fix grammar, spelling etc.
